app/client/src/components/InstallChaos/index.tsx

```diff
@@ -42,27 +68,93 @@ export function InstallChaos(props: InstallProps) {
 		setTimeout(() => setCopying(false), 3000);
 	}
 
+	function showYamlInPage(text: string) {
+		if (!viewing) {
+			fetchYamlAndShowInPage(yamlLink);
+			setViewing(true);
+		} else {
+			setYaml(`kubectl apply -f ${yamlLink}`);
```

Comment:
Can we use the constant definition from line no: 23 here?

Reply:
Yeah sure it will be replaced.

Reply:
Is that `const` used somewhere? Otherwise, you can remove it

Reply:
Yeah it is being used in 3 places to access the state. No changes have been made to its usage.

If i replace it with "let" it will be against the general rules of using state variables in React as they are made constant so as to allow a change only by their corresponding state setter methods.